src/NServiceBus.AcceptanceTests/Routing/MessageDrivenSubscriptions/When_subscribing_with_scaled_out_subscriber.cs

```diff
@@ -0,0 +1,77 @@
+namespace NServiceBus.AcceptanceTests.Routing.MessageDrivenSubscriptions
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using NUnit.Framework;
+    using Settings;
+
+    public class When_subscribing_with_scaled_out_subscriber : NServiceBusAcceptanceTest
```

Comment:
how does this test differentiate from `When_publishing_to_scaled_out_subscribers` ?